getting_started.rst

```diff
@@ -107,3 +109,27 @@ system.
   a signed social network. This demo calculates and shows structural imbalance for
   social networks of militant organization based on data from the Stanford Militants
   Mapping Project.
+
+.. _additional_tutorials:
+
+Additional Tutorials
+====================
+
+* `Getting Started with the D-Wave System <https://docs.dwavesys.com/docs/latest/doc_getting_started.html>`_
+
+  This guide in the
+  `System Documentation <https://docs.dwavesys.com/docs/latest/index.html>`_
+  introduces the D-Wave quantum computer, provides some key background information on
+  how the system works, and explains how to construct a simple problem that the system
+  can solve.
+
+* `D-Wave Problem-Solving Handbook <https://docs.dwavesys.com/docs/latest/doc_handbook.html>`_
```

Comment:
Could we simplify all the links (here and above) to the system docs with intersphinx?

Reply:
:std:doc:`Getting Started with the D-Wave System <sysdocs_gettingstarted:doc_getting_started>`